docs/index.rst

```diff
@@ -29,45 +29,32 @@ single-page web apps, and console apps.
        :align: center
        :width: 450px
 
-.. raw:: html
-
-  <div class="table">
 
 :ref:`Tutorial <tutorial>`
 --------------------------
 
-* :doc:`A quick Toga demonstration <tutorial/get-started>`
-* :doc:`A hands-on introduction to Toga </tutorial/index>`
-
-:ref:`Reference <reference>`
-----------------------------
-
-* :doc:`API reference <reference/api/index>`
-* :doc:`Toga's platform support <reference/platforms/index>`
-* :doc:`Widget support by platform <reference/widgets_by_platform>`
-* :doc:`Managing style with Toga <reference/style/index>`
-* :doc:`Toga's plugin interfaces <reference/plugins/index>`
+:doc:`A quick Toga demonstration <tutorial/get-started>` and a :doc:`a hands-on introduction to Toga </tutorial/index>`.
 
 :ref:`How-to <how-to>`
 ----------------------
 
-* :doc:`Topic guides </how-to/topics/index>`
-* :doc:`Contribute to Toga <how-to/contribute/index>`
-* :doc:`Internal Toga processes <how-to/internal/index>`
+Guides and recipes for common problems and tasks, including how to :doc:`contribute to Toga <how-to/contribute/index>`.
+
+:ref:`Topics <topic>`
+----------------------
```

Comment:
Title for these two should be "How-to guides" and "Topic guides". The underline length is also off.

On a bike shedding thing - I've just noticed this is in a "topic", not "topics" directory... I appreciate it's churn, but I think `topics` works better in the context (and given there's going to be a town crier config update needed on each repo anyway, maybe we can fix this at the same time)

Reply:
Agreed on "topics" - I went with the previously existing directory. Updated here. Will update it on Briefcase and Rubicon along with the towncrier config fix.